### DISCUSSION TITLE
Fatal Error

### DISCUSSION THREAD

**Discussion by jbgy** [YOURLS/YOURLS#3699] [Q&A]
## **Fatal error**

Incorrect DB config, or could not connect to DB
PDOException: could not find driver

I am running on nginx, since i host also another website.

**Comment by dgw** [comment]
Your error message indicates that PHP's `pdo_mysql` extension isn't available.

There is not enough information here to suggest a definite solution, but you most likely need to install another package or two from your OS distribution's repository, something like `php-mysql`.

**Reply by jbgy** [reply]
it seems i already have it installed

**Reply by dgw** [reply]
Maybe the extension is not enabled in PHP's configuration. Maybe you have the extension installed, but for a different PHP version than what nginx is sending requests to.

There are lots of possibilities, and you've still not provided any meaningful amount of information about your environment that would help someone help you troubleshoot.

**Reply by jbgy** [reply]
then please tell me what stats/infos you would need... im bad at providing infos....sorry
